### DISCUSSION TITLE
Will this driver ever be compatible with GTX 1080?

### DISCUSSION THREAD

**Discussion by Volian0** [NVIDIA/open-gpu-kernel-modules#323] [Q&A]
I installed the driver but it doesn't work for my card. Are there plans to support older GPUs?

**Comment by YusufKhan-gamedev** [comment]
https://github.com/NVIDIA/open-gpu-kernel-modules/issues/19

**Comment by Volian0** [comment]
oh ok then, I'm selling my card and buying AMD

**Reply by shmabler** [reply]
Yeah, me too, I'm selling my card and buying AMD. I have an EVGA 1080ti SC2 and it doesn't work well with Wayland and Fedora just crashed due to Nvidia Kernel Module while running Xorg. I've wanted better Linux support and might as well just switch to AMD.

**Reply by YusufKhan-gamedev** [reply]
You also get a more stable graphics card that'l last more than 10 years.
